# Number of channels
uint8 channels

# Sampling rate [Hz]
uint32 sample_rate

# Audio format (e.g. int16, float32)
string sample_format

# Audio coding format (e.g. wav, mp3)
string coding_format